MdeModulePkg/Include/Guid/AcpiTableHob.h

```diff
@@ -0,0 +1,20 @@
+/** @file
+ Define the GUID gPldAcpiTableGuid HOB struct.
+
+Copyright (c) 2021, Intel Corporation. All rights reserved.<BR>
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _ACPI_TABLE_HOB_H_
+#define _ACPI_TABLE_HOB_H_
+
+#include <Uefi.h>
+
+typedef struct {
+  EFI_PHYSICAL_ADDRESS          Rsdp;

```

Comment:
Similar comments as that for SmBiosTableHob.h regarding EFI dependency.

Reply:
I remenber Ray has a concern that "EFI_PHYSICAL_ADDRESS" looks like it is UEFI related, while our spec should be more independent.